easy-coding-standard.yml

```diff
@@ -1,23 +1,9 @@
-imports:
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/clean-code.yaml' }
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/common.yaml' }
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/php70.yaml' }
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/php71.yaml' }
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/psr12.yaml' }
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/symfony.yaml' }
-    - { resource: 'vendor/symplify/easy-coding-standard/config/set/symfony-risky.yaml' }
-
 services:
     # most of these services are taken from symplify.yml
     # see https://github.com/Symplify/Symplify/blob/master/ecs.yml
 
-    # PHP 5.5
-    Symplify\CodingStandard\Fixer\Php\ClassStringToClassConstantFixer: ~
-
     # Control Structures
-    Symplify\CodingStandard\Fixer\Property\ArrayPropertyDefaultValueFixer: ~
     Symplify\CodingStandard\Fixer\ArrayNotation\StandaloneLineInMultilineArrayFixer: ~
-    Symplify\CodingStandard\Fixer\ControlStructure\RequireFollowedByAbsolutePathFixer: ~
```

Comment:
These 3 classes do not exist anymore.

If you need it, there are more reliable Rector replacements:

- https://tomasvotruba.com/blog/2020/05/04/how-to-upgrade-to-symplify-8-from-sniffs-to-phpstan-rules/
- https://tomasvotruba.com/blog/2020/05/11/how-to-upgrade-to-symplify-8-from-fixers-to-rector-rules/